content/guides/models/track/runs/_index.md

```diff
@@ -386,7 +386,7 @@ Use the **Overview** tab to learn about specific run information in a project, s
 * **System hardware**: The hardware W&B uses to compute the run.
 * **Tags**: A list of strings. Tags are useful for organizing related runs together or applying temporary labels like `baseline` or `production`.
 * **W&B CLI version**: The W&B CLI version installed on the machine that hosted the run command.
-<!-- * **Git state**: -->
+* **Git state**: The most recent ("last") git commit SHA of a repository or working directory where the run is initialized. This field is empty if you do not enable git when you create the run or if the git information is not available.
```

Comment:
```suggestion
* **Git state**: The most recent ("last") git commit SHA of a repository or working directory where the run is initialized. This field is empty if you do not enable Git when you create the run or if the Git information is not available.
```
Not sure you neecd both most recent and "last" -- is the latter what the field actually is called in the output or something?